src/service/xhr-impl.js

```diff
@@ -195,9 +176,11 @@ export class Xhr {
    *
    * @param {string} input
    * @param {?FetchInitDef=} opt_init
+   * @param {boolean=} opt_skipProcessing Disables any processing of the
+   *     response, including asserting the response is 2XX.
    * @return {!Promise<!JSONType>}
    */
-  fetchJson(input, opt_init) {
+  fetchJson(input, opt_init, opt_skipProcessing) {
```

Comment:
IMHO, `fetch` itself should always return non-asserted response. The closer we are to fetch API for `fetch` method specifically - the better. `fetchJson` is a different story a bit, since there's no point to parse JSON when there's no content there. But overall, I also see value getting closer to the `fetch` API.